### PR TITLE
Feature/ui

### PR DESCRIPTION
**Pull request by jeanjoe**
Hello @joelethan @SokoPaulSokool @okellogabrielinnocent @walimike
Help Review my PR for my UIs

### CODE REVIEW

**Comment by KicaRonaldOkello** [comment]
Keep up with the atomic commits. I think the work is ready to merge

**Review by kaggwachristopher — APPROVED** [review]
The code is okay and the responsiveness of your site....you just need to add a few features on the admin page but its very ok to merge as per now

**Comment by sekayasin** [comment]
@jeanjoe bravo.., a few pointers here.. i think the admin link to be visible by everyone is not good design.. usually admins have a special url known to them only..

**Comment by jeanjoe** [comment]
Thanks @all for your comments, @sekayasin  I wanted to make it easily accessible in this context otherwise I thought about that as well.

**Review by okellogabrielinnocent — APPROVED** [review]
Good work here when switching between admin and user pages

File: UI/sign-up.html [hunk 1]
```diff
@@ -38,7 +40,7 @@
             <input type="password" name="confirm_password" value="" placeholder="******" min="6">
           </div>
           <div class="button">
-            <button type="submit" name="signup">Sign Up</button>
+            <button type="submit" class="submit">Sign Up</button>
```
Comment (kdaglas) [inline UI/sign-up.html]: i have not yet understood this... i think you can make one button tag and put all the attributes above together

Comment (jeanjoe) [reply UI/sign-up.html]: @kdaglas  thanks for your concern. The later is a modification of the former. I was using a submit button, but I needed to keep the flow of the page when a user clicks Sign up button that's why I changed to anchor from button.

**Comment by mozzy22** [comment]
cool work @jeanjoe , just improve a little bit onthe responsveness

**Comment by Luleherll** [comment]
your structure is good

**Comment by SokoPaulSokool** [comment]
Good work. though when someone visits your page it signs them in automatically. Are you planning to first load the login page?

**Comment by wasibani-roy** [comment]
Looking good the responsiveness was a good touch

**Comment by mozzy22** [comment]
look cool

**Comment by walimike** [comment]
@jeanjoe this is nice, but probably youy could clarify on this, commit messages are supposed to be in present continous tense forinstance  adds font-family etc

**Comment by walimike** [comment]
Hey, also, on your order.html you mis spelled "penging" and put "pendid" so please correct that

**Comment by walimike** [comment]
pending*

**Comment by kyakusahmed** [comment]
i love your work

**Comment by joelethan** [comment]
Good work @jeanjoe , i think you are good to go...

**Pull request merged**